inc/class-provider.php

```diff
@@ -90,13 +91,24 @@ protected function request( array $args ): MediaList {
 			) );
 		}
 
+		$total = absint( $response->get_headers()['x-wp-total'] );
```

Comment:
Can we sure this is always there? Or do we want to ensure this doesn't break?

```suggestion
		$total = absint( $response->get_headers()['x-wp-total'] ?? 0 );
```